hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/security/token/TestZKDelegationTokenSecretManagerImpl.java

```diff
@@ -0,0 +1,216 @@
+package org.apache.hadoop.hdfs.server.federation.security.token;
```

Comment:
lack of license annotation？

Reply:
Sigh.. somehow I missed it. Thanks for pointing out.